Dockerfile

```diff
@@ -17,9 +17,7 @@ RUN chown -R deno:deno /bot
 USER deno
 
 # Cache all of the dependencies so they don't need to be downloaded every run
-RUN cp /bot/configs.example.ts /bot/configs.ts && \
-    deno cache deps.ts && \
-    deno cache mod.ts
+RUN deno cache mod.ts
 
 # Finally run the bot
-CMD ["run", "--allow-net", "--allow-read=/bot", "--allow-write=/bot", "./mod.ts"]
+CMD ["run", "--allow-net", "--allow-read=/bot", "--allow-write=/bot", "--allow-env", "./mod.ts"]
```

Comment:
Discuss: Is it worth changing this to `-A`

Reply:
To be honest, `--allow-all` just defeats the purpose of Deno

Reply:
I don't see any benefits from this personally and i see a lot of issues/harm that could come from it